README.md

```diff
@@ -6,36 +6,42 @@
 
 # Fulcio
 
-_A New Kind of Root CA For Code Signing_
+_A Free-to-Use CA For Code Signing_
 
-fulcio is a free Root-CA for code signing certs - issuing certificates based on an OIDC email address.
+Fulcio is a free-to-use certificate authority for issuing code signing certificates
+for an OpenID Connect (OIDC) identity, such as email address.
 
-fulcio only signs short-lived certificates that are valid for under 20 minutes.
+Fulcio only issues short-lived certificates that are valid for 10 minutes.
 
 ## Status
 
-Fulcio is a *work in progress*!
+Fulcio is in General Availability, offering a 99.5 Availability SLO.
```

Comment:
nit: fulcio.sigstore.dev is the only thing we offer an SLO on, we might want to try to be clearer around the project and code (can be run by anyone) vs the public good instance the community runs

Reply:
we should also explicitly acknowledge that we intend to adhere to https://semver.org/ rules around API stability